qa/rpc-tests/test_framework/authproxy.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, service_url, service_name=None, timeout=HTTP_TIMEOUT, connect
                                                   timeout)
         else:
             self.__conn = httplib.HTTPConnection(self.__url.hostname, port,
-                                                 False, timeout)
+                                                 timeout=timeout)
```

Comment:
Hm @luke-jr's commit seems to be more thorough. I indeed didn't test the https path. Will replace with his commit.